tools/tiller-hack/tiller-hack.go

```diff
@@ -0,0 +1,232 @@
+package main
+
+import (
+	"bytes"
+	"compress/gzip"
+	"encoding/base64"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/sirupsen/logrus"
+
+	// "gopkg.in/yaml.v2"
+	// "github.com/ghodss/yaml"
+	"k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/client-go/util/retry"
+
+	"k8s.io/apimachinery/pkg/labels"
+
+	"github.com/golang/protobuf/proto"
+	"k8s.io/helm/pkg/proto/hapi/chart"
+	rspb "k8s.io/helm/pkg/proto/hapi/release"
+)
+
+var (
+	kubeconfig         = flag.String("kubeconfig", "", "Absolute path to the kubeconfig file.")
+	configMapName      = flag.String("configmap", "", "Name of the configmap to patch.")
+	configMapNamespace = flag.String("namespace", "kube-system", "Namespace of the configmap, defaults to \"kube-system\"")
+	restartTillerFlag  = flag.Bool("restart-tiller", false, "Restart tiller after patching the configmap")
+)
+
+var b64 = base64.StdEncoding
```

Comment:
i'd suggest dropping this shortcut as it's not pervasive and makes it harder to follow the code without looking at this definition.

Reply:
done